### PR TITLE
Add toInclude

### LINKED ISSUE #41: New Matcher: .toInclude(substring)
**Feature Request**

### .toInclude(substring)

Use `.toInclude` when checking if a `String` includes the given `String` substring.

```js
test('passes when value includes substring', () => {
  expect('hello world').toInclude('ell');
  expect('hello world').not.toInclude('bob');
});
```

### PR DESCRIPTION
**Pull request by grug**
### What

Add `toInclude` matcher.

### Why

Closes #41 

### Housekeeping

- [X] Unit tests
- [X] Documentation is up to date
- [X] No additional lint warnings
- [X] Add yourself to contributors list (`yarn contributor`)

### CODE REVIEW

**Comment by mattphillips** [comment]
Looks good @grug just need to rebase master back into your branch :smile:

**Comment by grug** [comment]
Awesome. I think I've rebased that correctly. Let me know if you need anything else for this PR to be merged :)

File: src/matchers/toInclude/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = (actual, expected) => () => {
```
Comment (mattphillips) [inline src/matchers/toInclude/index.js]: Just noticed this function isn't returning the message if you remove the curly braces it should fix this snapshot:

```
exports[`.toInclude .not.toInclude fails when a string does have a given substring 1`] = `"<red>No message was specified for this matcher.</>"`;
```

```js
const passMessage = (actual, expected) => () =>
  matcherHint('.not.toInclude') +
    '\n\n' +
    'Expected string to not include:\n' +
    `  ${printExpected(expected)}\n` +
    'Received:\n' +
    `  ${printReceived(actual)}`;
```

Comment (grug) [reply src/matchers/toInclude/index.js]: Yup I'll fix that now

**Comment by grug** [comment]
Okay - I've fixed up the changes you've suggested :) let me know if I should do anything else before this can be merged.

**Comment by mattphillips** [comment]
Good job @grug thanks!

**Pull request merged**